From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 02:11:02 +0100
Subject: Add some comments to mta.te - droppable?

---
 policy/modules/services/mta.te |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/policy/modules/services/mta.te b/policy/modules/services/mta.te
index 64268e4..7878ff1 100644
--- a/policy/modules/services/mta.te
+++ b/policy/modules/services/mta.te
@@ -5,11 +5,18 @@ policy_module(mta, 2.3.0)
 # Declarations
 #
 
+# attribute used for domains that act on behalf of the user to deliver mail
+# to the queue
 attribute mailcontent_type;
 attribute mta_exec_type;
 attribute mta_user_agent;
+
+# attribute used for domains that deliver mail locally
 attribute mailserver_delivery;
+
 attribute mailserver_domain;
+
+# attribute used for domains that send mail externally (smtp or lmtp)
 attribute mailserver_sender;
 
 attribute user_mail_domain;
