From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sat, 18 Feb 2012 01:39:09 +0100
Subject: disable user-based access control

---
 build.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/build.conf b/build.conf
index 5a521c4..5113c1b 100644
--- a/build.conf
+++ b/build.conf
@@ -50,7 +50,7 @@ MONOLITHIC = n
 
 # User-based access control (UBAC)
 # Enable UBAC for role separations.
-UBAC = y
+UBAC = n
 
 # Custom build options.  This field enables custom
 # build options.  Putting foo here will enable
