From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Mon, 27 Feb 2012 00:17:47 +0100
Subject: Allow users read access to /var/lib/apt/lists

---
 policy/modules/system/userdomain.if |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/policy/modules/system/userdomain.if b/policy/modules/system/userdomain.if
index 492703f..5225bbd 100644
--- a/policy/modules/system/userdomain.if
+++ b/policy/modules/system/userdomain.if
@@ -123,6 +123,13 @@ template(`userdom_base_user_template',`
 		fs_manage_dos_files($1_t)
 	')
 
+	ifdef(`distro_debian', `
+		# allow reading /var/lib/apt/lists
+		apt_read_db($1_t)
+		# allow reading /var/cache/apt - should not be needed but
+		# does not really matter
+		apt_read_cache($1_t)
+	')
 ')
 
 #######################################
