From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Fri, 2 Mar 2012 03:13:10 +0100
Subject: Allow inetd_t setrlimit access.

---
 policy/modules/services/inetd.te |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/policy/modules/services/inetd.te b/policy/modules/services/inetd.te
index c51a7b2..1c2f387 100644
--- a/policy/modules/services/inetd.te
+++ b/policy/modules/services/inetd.te
@@ -40,7 +40,7 @@ ifdef(`enable_mcs',`
 
 allow inetd_t self:capability { setuid setgid };
 dontaudit inetd_t self:capability sys_tty_config;
-allow inetd_t self:process { setsched setexec };
+allow inetd_t self:process { setrlimit setsched setexec };
 allow inetd_t self:fifo_file rw_fifo_file_perms;
 allow inetd_t self:tcp_socket create_stream_socket_perms;
 allow inetd_t self:udp_socket create_socket_perms;
@@ -77,6 +77,7 @@ corenet_tcp_bind_generic_node(inetd_t)
 corenet_udp_bind_generic_node(inetd_t)
 corenet_tcp_connect_all_ports(inetd_t)
 corenet_sendrecv_all_client_packets(inetd_t)
+allow inetd_t self:netlink_route_socket r_netlink_socket_perms;
 
 # listen on service ports:
 corenet_tcp_bind_amanda_port(inetd_t)
