From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Fri, 2 Mar 2012 03:24:39 +0100
Subject: Allow network manager to run wpa_cli_exec_t programs.

---
 policy/modules/services/networkmanager.te |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/services/networkmanager.te b/policy/modules/services/networkmanager.te
index 0619395..8bf1bca 100644
--- a/policy/modules/services/networkmanager.te
+++ b/policy/modules/services/networkmanager.te
@@ -270,6 +270,7 @@ optional_policy(`
 # wpa_cli local policy
 #
 
+domain_auto_trans(NetworkManager_t, wpa_cli_exec_t, wpa_cli_t)
 allow wpa_cli_t self:capability dac_override;
 allow wpa_cli_t self:unix_dgram_socket create_socket_perms;
 
