From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 00:10:16 +0100
Subject: Alsa debian locations

---
 policy/modules/admin/alsa.fc |   14 ++++++++++----
 1 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/policy/modules/admin/alsa.fc b/policy/modules/admin/alsa.fc
index d362d9c..20062d1 100644
--- a/policy/modules/admin/alsa.fc
+++ b/policy/modules/admin/alsa.fc
@@ -2,10 +2,16 @@ HOME_DIR/\.asoundrc	--	gen_context(system_u:object_r:alsa_home_t,s0)
 
 /bin/alsaunmute		--	gen_context(system_u:object_r:alsa_exec_t,s0)
 
-/etc/alsa/asound\.state --	gen_context(system_u:object_r:alsa_etc_rw_t,s0)
-/etc/alsa/pcm(/.*)?		gen_context(system_u:object_r:alsa_etc_rw_t,s0)
-/etc/asound(/.*)?		gen_context(system_u:object_r:alsa_etc_rw_t,s0)
-/etc/asound\.state	--	gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+/etc/alsa/pcm(/.*)?	        gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+/etc/asound(/.*)?               gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+ifdef(`distro_debian', `
+/var/lib/alsa/asound\.state --	gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+/usr/share/alsa/alsa\.conf      gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+/usr/share/alsa/pcm(/.*)?       gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+', `
+/etc/alsa/asound\.state --      gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+/etc/asound\.state      --       gen_context(system_u:object_r:alsa_etc_rw_t,s0)
+')
 
 /sbin/alsactl 		--	gen_context(system_u:object_r:alsa_exec_t,s0)
 /sbin/salsa 		--	gen_context(system_u:object_r:alsa_exec_t,s0)
