From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 00:14:45 +0100
Subject: Correctly label rotated logs of apt

---
 policy/modules/admin/apt.fc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/policy/modules/admin/apt.fc b/policy/modules/admin/apt.fc
index e4f4850..379ebfb 100644
--- a/policy/modules/admin/apt.fc
+++ b/policy/modules/admin/apt.fc
@@ -15,7 +15,7 @@
 # aptitude lock
 /var/lock/aptitude			gen_context(system_u:object_r:apt_lock_t,s0)
 # aptitude log
-/var/log/aptitude			gen_context(system_u:object_r:apt_var_log_t,s0)
+/var/log/aptitude.*			gen_context(system_u:object_r:apt_var_log_t,s0)
 
 # dpkg terminal log
 /var/log/apt(/.*)?			gen_context(system_u:object_r:apt_var_log_t,s0)
