From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 00:50:54 +0100
Subject: Set the context of /var/run/portmap_mapping

---
 policy/modules/services/portmap.fc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/services/portmap.fc b/policy/modules/services/portmap.fc
index 76f5834..2a46efb 100644
--- a/policy/modules/services/portmap.fc
+++ b/policy/modules/services/portmap.fc
@@ -4,6 +4,7 @@
 ifdef(`distro_debian',`
 /sbin/pmap_dump		--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
 /sbin/pmap_set		--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
+/var/run/portmap_mapping --	gen_context(system_u:object_r:portmap_var_run_t,s0)
 ', `
 /usr/sbin/pmap_dump	--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
 /usr/sbin/pmap_set	--	gen_context(system_u:object_r:portmap_helper_exec_t,s0)
