From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 01:06:09 +0100
Subject: Allow reading and writing tmpfs files while relabelling

---
 policy/modules/kernel/devices.if |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/policy/modules/kernel/devices.if b/policy/modules/kernel/devices.if
index c23c3b5..ce92153 100644
--- a/policy/modules/kernel/devices.if
+++ b/policy/modules/kernel/devices.if
@@ -140,7 +140,7 @@ interface(`dev_mounton',`
 interface(`dev_relabel_all_dev_nodes',`
 	gen_require(`
 		attribute device_node;
-		type device_t;
+		type device_t, tmpfs_t;
 	')
 
 	relabelfrom_dirs_pattern($1, device_t, device_node)
@@ -150,6 +150,7 @@ interface(`dev_relabel_all_dev_nodes',`
 	relabelfrom_sock_files_pattern($1, device_t, device_node)
 	relabel_blk_files_pattern($1, device_t, { device_t device_node })
 	relabel_chr_files_pattern($1, device_t, { device_t device_node })
+	allow $1 tmpfs_t:chr_file { read write };
 ')
 
 ########################################
