From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 00:35:04 +0100
Subject: Allow mono_t to be in role unconfined_r Closes: #540143

---
 policy/modules/apps/mono.te |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/apps/mono.te b/policy/modules/apps/mono.te
index dff0f12..27f7471 100644
--- a/policy/modules/apps/mono.te
+++ b/policy/modules/apps/mono.te
@@ -45,6 +45,7 @@ optional_policy(`
 	unconfined_domain(mono_t)
 	unconfined_dbus_chat(mono_t)
 	unconfined_dbus_connect(mono_t)
+	in_unconfined_r(mono_t)
 ')
 
 optional_policy(`
