From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 02:42:22 +0100
Subject: Label /run/mdadm/map Closes: #643490

---
 policy/modules/system/raid.fc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/system/raid.fc b/policy/modules/system/raid.fc
index ed9c70d..392ffa1 100644
--- a/policy/modules/system/raid.fc
+++ b/policy/modules/system/raid.fc
@@ -1,4 +1,5 @@
 /dev/.mdadm.map		--	gen_context(system_u:object_r:mdadm_map_t,s0)
+/run/mdadm/map		--	gen_context(system_u:object_r:mdadm_map_t,s0)
 
 /sbin/mdadm		--	gen_context(system_u:object_r:mdadm_exec_t,s0)
 /sbin/mdmpd		--	gen_context(system_u:object_r:mdadm_exec_t,s0)
