From: =?UTF-8?q?Mika=20Pfl=C3=BCger?= <debian@mikapflueger.de>
Date: Sun, 4 Mar 2012 03:04:09 +0100
Subject: droppable reordering of udev permissions

---
 policy/modules/system/udev.te |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/policy/modules/system/udev.te b/policy/modules/system/udev.te
index eedca43..2b17d17 100644
--- a/policy/modules/system/udev.te
+++ b/policy/modules/system/udev.te
@@ -171,12 +171,12 @@ seutil_read_file_contexts(udev_t)
 seutil_domtrans_setfiles(udev_t)
 
 sysnet_domtrans_ifconfig(udev_t)
+sysnet_manage_config(udev_t)
 sysnet_domtrans_dhcpc(udev_t)
 sysnet_rw_dhcp_config(udev_t)
 sysnet_read_dhcpc_pid(udev_t)
 sysnet_delete_dhcpc_pid(udev_t)
 sysnet_signal_dhcpc(udev_t)
-sysnet_manage_config(udev_t)
 sysnet_etc_filetrans_config(udev_t)
 
 userdom_dontaudit_search_user_home_content(udev_t)
