From: Russell Coker <russell@coker.com.au>
Date: Sun, 14 Oct 2012 23:55:32 +0200
Subject: Label postalias as postfix_master_exec_t for newaliases

---
 policy/modules/services/postfix.fc |    1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/services/postfix.fc b/policy/modules/services/postfix.fc
index 614ca4f..2de1047 100644
--- a/policy/modules/services/postfix.fc
+++ b/policy/modules/services/postfix.fc
@@ -34,6 +34,7 @@ ifdef(`distro_redhat', `
 ')
 /etc/postfix/postfix-script.* -- gen_context(system_u:object_r:postfix_exec_t,s0)
 /etc/postfix/prng_exch	--	gen_context(system_u:object_r:postfix_prng_t,s0)
+/usr/sbin/postalias	--	gen_context(system_u:object_r:postfix_master_exec_t,s0)
 /usr/sbin/postcat	--	gen_context(system_u:object_r:postfix_master_exec_t,s0)
 /usr/sbin/postdrop	--	gen_context(system_u:object_r:postfix_postdrop_exec_t,s0)
 /usr/sbin/postfix	--	gen_context(system_u:object_r:postfix_master_exec_t,s0)
