From: Russell Coker <russell@coker.com.au>
Date: Sun, 21 Oct 2012 03:41:07 +0200
Subject: Label /sbin/xtables-multi (the new iptables)

---
 policy/modules/system/iptables.fc |    1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/iptables.fc b/policy/modules/system/iptables.fc
index 14f3c5e..ef9762d 100644
--- a/policy/modules/system/iptables.fc
+++ b/policy/modules/system/iptables.fc
@@ -15,6 +15,7 @@
 /sbin/ipvsadm			--	gen_context(system_u:object_r:iptables_exec_t,s0)
 /sbin/ipvsadm-restore		--	gen_context(system_u:object_r:iptables_exec_t,s0)
 /sbin/ipvsadm-save		--	gen_context(system_u:object_r:iptables_exec_t,s0)
+/sbin/xtables-multi		--	gen_context(system_u:object_r:iptables_exec_t,s0)
 
 /usr/sbin/ipchains.*		--	gen_context(system_u:object_r:iptables_exec_t,s0)
 /usr/sbin/iptables		--	gen_context(system_u:object_r:iptables_exec_t,s0)
